### PR TITLE
[WIP] update orthography profile

### PR DESCRIPTION
**Pull request by Wu-Urbanek**
X and H in Ratliff's data are rising and falling tones. 
Please give suggestions on how to mark those cases.

### CODE REVIEW

**Comment by LinguList** [comment]
rising tone is prototypically ¹⁵ and falling is ⁵¹, so X/¹⁵ and H/⁵¹ right?

**Comment by Wu-Urbanek** [comment]
ah good, I will change that.

**Comment by Wu-Urbanek** [comment]
first version of the orthography profile is done :) :)

File: TRANSCRIPTION.md [hunk 1]
```diff
@@ -4,9 +4,149 @@
...
+| i̯ | 4 | ✓ | ✓ |
+| sʰ | 4 | ✓ | ✓ |
+| tɕʰ | 4 | ✓ | ✓ |
+| ² | 4 | ✓ | ✓ |
+| ²¹³ | 4 | ✓ | ✓ |
+| ɭ | 4 | ✓ | ✓ |
+| ʃ | 4 | ✓ | ✓ |
+| θʰ | 4 | ✓ | ✓ |
+| ⁸/⁴⁴ | 4 | ✓ | ✓ |
+| mʐ | 3 | ✓ | ✓ |
```
Comment (LinguList) [inline TRANSCRIPTION.md:103]: this is not a valid segment, Mei-Shin, it should resolve to `m ʐ`

File: TRANSCRIPTION.md [hunk 2]
```diff
@@ -4,9 +4,149 @@
...
+| ɕʰ | 3 | ✓ | ✓ |
+| ɖʐ | 3 | ✓ | ✓ |
+| ɳ | 3 | ✓ | ✓ |
+| ˀw | 3 | ✓ | ✓ |
+| ⁵⁴ | 3 | ✓ | ✓ |
+| kʷ | 2 | ✓ | ✓ |
+| l̥ʰ | 2 | ✓ | ✓ |
+| oː | 2 | ✓ | ✓ |
+| pʐ | 2 | ✓ | ✓ |
+| tʂ | 2 | ✓ | ✓ |
```
Comment (LinguList) [inline TRANSCRIPTION.md:115]: tʂ is not correct in terms of BIPA, please paste it [here](http://digling.org/calc/clts/) to get the right version.

Comment (LinguList) [reply TRANSCRIPTION.md:115]: pʐ should be `p ʐ`

File: TRANSCRIPTION.md [hunk 3]
```diff
@@ -4,9 +4,149 @@
...
+| ɔː | 2 | ✓ | ✓ |
+| ɖ | 2 | ✓ | ✓ |
+| ɛː | 2 | ✓ | ✓ |
+| ʈʂʰ | 2 | ✓ | ✓ |
+| ʨ | 2 | ✓ | ✓ |
+| cʰ | 1 | ✓ | ✓ |
+| eː | 1 | ✓ | ✓ |
+| pʰ | 1 | ✓ | ✓ |
+| tsʷ | 1 | ✓ | ✓ |
+| tʂʰ | 1 | ✓ | ✓ |
```
Comment (LinguList) [inline TRANSCRIPTION.md:130]: check with the BIPA/CLTS browser, link is shared above.

File: TRANSCRIPTION.md [hunk 4]
```diff
@@ -4,9 +4,149 @@
...
+| ʈ | 10 | ✓ | ✓ |
+| θ | 10 | ✓ | ✓ |
+| ⁵⁴⁵ | 9 | ✓ | ✓ |
+| g | 8 | ✓ | ✓ |
+| ²²¹ | 8 | ✓ | ✓ |
+| ɤ | 8 | ✓ | ✓ |
+| ɯ | 8 | ✓ | ✓ |
+| ⁵ | 8 | ✓ | ✓ |
+| tʃ | 7 | ✓ | ✓ |
+| u̯ | 7 | ✓ | ✓ |
```
Comment (LinguList) [inline TRANSCRIPTION.md:73]: u̯ is `w`

File: TRANSCRIPTION.md [hunk 5]
```diff
@@ -4,9 +4,149 @@
...
+| c | 5 | ✓ | ✓ |
+| l̥ | 5 | ✓ | ✓ |
+| ð | 5 | ✓ | ✓ |
+| ɐ | 5 | ✓ | ✓ |
+| ɔ̃ | 5 | ✓ | ✓ |
+| ɬ | 5 | ✓ | ✓ |
+| ʉ | 5 | ✓ | ✓ |
+| ⁴ | 5 | ✓ | ✓ |
+| ⁷/³³ | 5 | ✓ | ✓ |
+| i̯ | 4 | ✓ | ✓ |
```
Comment (LinguList) [inline TRANSCRIPTION.md:94]: `i̯` is `j`

**Review by LinguList — COMMENTED** [review]
not bad, but please see my recommendations. Note also that you should make a check similar to the one I made for chenhmongmien, by adding a wanghmongmiencommands folder, to see that the structures is matching our expectations. So far, there are still some problems.

**Pull request merged**